lib/sycamore/sycamore/utils/pdf.py

```diff
@@ -119,3 +123,112 @@ def convert_from_path_streamed_batched(filename: str, batch_size: int) -> Genera
 
     if len(batch) > 0:
         yield batch
+
+
+def pdf_to_image_files(pdf_path: str, file_dir: Path) -> Generator[Path, None, None]:
+    """Writes the files (streamed) into file_dir.  Caller is responsible for calling
```

Comment:
Should we have a similar note warning that the Aryn Partitioning Service will call this function?